code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -623,29 +623,29 @@
 
 //Code to handle finding and nomming nearby food items
 /mob/living/simple_animal/proc/handle_foodscanning()
-	if (client || !hunger_enabled || !autoseek_food)
+	if (client || !autoseek_food)
```

Comment:
did you mean to remove !hunger_enabled

Reply:
Does this not make them constantly hungry, which means they'll always seek food?

It feels like it could have the knock on of more player corpse gibbings.

Reply:
I moved the hunger_enabled check to the life proc so its not neededlessly doble checking